src/templates/Navigator/MainNavigatorComponent/Readme.md

```diff
@@ -0,0 +1,5 @@
+MainNavigatorComponent Example:
+
+    <MainNavigatorComponent>
```

Comment:
@j-clark Yeah... I started off doing that, but then I thought it would be confusing, since it's really only to be used with the Navigator as a wrapping component... what do you think about `NavigatorMain` and `NavigatorSidebar`?